src/dials/algorithms/indexing/bravais_settings.py

```diff
@@ -89,6 +89,12 @@ def __init__(self, other):
 
 
 class RefinedSettingsList(list):
+    def __init__(self, *args, cb_op_to_primitive=None, **kwargs):
+        if cb_op_to_primitive is None:
+            cb_op_to_primitive = sgtbx.change_of_basis_op()
+        self.cb_op_to_primitive = cb_op_to_primitive
```

Comment:
This warning is perhaps legitimate? Is there a way we can trigger this code in the regression tests to ensure that this important fix remains fixed?

Reply:
This was just backwards compatibility for consistency with the previous function signature - in practice, at least as called within dials code, `cb_op_to_primitive` is never `None`. The actual fix is covered by the test in `test_refine_bravais_settings_non_primitive_input`.